### ISSUE TITLE
Send a message in trial mode. Error Identification

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1463]
## Description of issue

Errors identified late in the process

When sending a message in trial mode, a user can only send messages to themselves and others on their team, however this is communicated two steps after entering the email. On the Add recipients page, instructions appear that inform the user that the message can only be sent to GC Notify team members, however the input is not validated. If a user enters an email address that does not belong to a team member, they are still able to proceed to the next page to review the message before sending. The “You cannot send to this email address” error message only appears when they attempt to send the message. Consider showing this error at the first step, when the email is entered. 

<img width="1327" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd3VZQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f59539830fe23dabe61657b95935e5d24b685773/image.png" alt="image.png" />

## Potential fix

- Validate email with trial mode rules, when in trial.
